posts/archive/ansible-servicenow-opening-and-closing-tickets.md

```diff
@@ -0,0 +1,218 @@
+sizes="(max-width: 1999px) 100vw, 1999px"}\
+
+Notice that the short description has been filled out by our Ansible
+Playbook task: This is a test opened by Ansible as well as the priority
+`2 - High`.
+
+![blog_servicenow-screen-2](https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=1999&name=blog_servicenow-screen-2.png){style="width: 1999px;"
+width="1999"
+srcset="https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=1000&name=blog_servicenow-screen-2.png 1000w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=1999&name=blog_servicenow-screen-2.png 1999w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=2999&name=blog_servicenow-screen-2.png 2999w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=3998&name=blog_servicenow-screen-2.png 3998w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=4998&name=blog_servicenow-screen-2.png 4998w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_servicenow-screen-2.png?width=5997&name=blog_servicenow-screen-2.png 5997w"
+sizes="(max-width: 1999px) 100vw, 1999px"}\
+
+## Closing a ServiceNow ticket
+
+Now that we've demonstrated the opening of ServiceNow tickets, we should
+demonstrate closing or resolving the ticket as well. This is done by
+specifying the `state`
+parameter in another Ansible task. This is where it can get tricky
+because `state` is a
+parameter of the record dictionary as well as a parameter of the
+`snow_record` module.
+Please be mindful of this multi-purpose parameter used in Ansible.
+
+The following is a snippet from the
+`record` dictionary when
+we created our ticket:
+
+![blog_closing-servicenow-ticket](https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=914&name=blog_closing-servicenow-ticket.png){style="width: 914px;"
+width="914"
+srcset="https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=457&name=blog_closing-servicenow-ticket.png 457w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=914&name=blog_closing-servicenow-ticket.png 914w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=1371&name=blog_closing-servicenow-ticket.png 1371w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=1828&name=blog_closing-servicenow-ticket.png 1828w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=2285&name=blog_closing-servicenow-ticket.png 2285w, https://www.ansible.com/hs-fs/hubfs/Images/blog-social/blog_closing-servicenow-ticket.png?width=2742&name=blog_closing-servicenow-ticket.png 2742w"
```

Comment:
hubspot specific links, display broken. 
we have to search for `/hs-fs/` and `hubfs/`, as those link to hubspot assets.